webservices/tasks/refresh_db.py

```diff
@@ -9,20 +9,24 @@
 
 logger = logging.getLogger(__name__)
 
+SLACK_BOTS = "#bots"
+SLACK_ALERTS = "#alerts"
+
 
 @app.task
 def refresh_materialized_views():
     """
     Refresh public materialized views.
     """
-    manage.logger.info("Starting daily update materialized views...")
+    manage.logger.info(" Starting daily update materialized views...")
     try:
         manage.refresh_materialized()
         download.clear_bucket()
-        slack_message = "*Success* daily update materialized views for {0} completed".format(get_app_name())
-        utils.post_to_slack(slack_message, '#bots')
+        slack_message = "*Success* daily update materialized views for {0} completed.".format(get_app_name())
+        utils.post_to_slack(slack_message, SLACK_BOTS)
         manage.logger.info(slack_message)
     except Exception as error:
         manage.logger.exception(error)
-        slack_message = "*ERROR* daily update materialized views failed for {0}. Check logs.".format(get_app_name())
-        utils.post_to_slack(slack_message, "#bots")
+        slack_message = "*ERROR* daily update materialized views failed for {0}.".format(get_app_name())
+        slack_message = slack_message + "\n Error message: " + str(error)
+        utils.post_to_slack(slack_message, SLACK_ALERTS)
```

Comment:
Thanks for your work on this, @fec-jli! Since these messages go in `#bots` when all is well and `#alerts` when they fail, it might be helpful to post in both channels when they fail. What do you think? Happy to discuss in Slack!

Reply:
@lbeaufort #bots is too generic and it's not that easy to find the scheduled jobs related messages and going back and forth btw #bots and #alerts is more annoying . It would be better to send both success and failure messages to one channel ie #alerts  or create a seperate channel and post all schedule jobs related msgs there. Makes our lives easy to debug when we really need it.
cc @fec-jli

Reply:
Thanks @lbeaufort @pkfec review my PR and good suggestion.

Reply:
👍 looks good, thanks for the discussion earlier @fec-jli and @pkfec!